scripts/convert.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * This js file is used to convert `./reports/registries.json` generated by data generator to `shared` files.
+ * Put the file in the same directory as the js file.
+ * Use `node ./convert.js` to execute.
+ * 
+ * @author SPGoding
+ */
+
+const fs = require('fs')
+
+const registriesJson = JSON.parse(fs.readFileSync('./registries.json', { encoding: 'utf8' }))
+
+function convert(keyName, fileName) {
+    const arr = []
+    let ans = `{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://raw.githubusercontent.com/Levertion/minecraft-json-schema/master/java/shared/${fileName}.json",
+    "type": "string",
+    "enum": [

```

Comment:
Why not just use the build in `JSON.stringify` and create an Object that way instead? That would be much more maintainable.

Reply:
That's my fault! Fixed in 70618e5.